app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingSettingsViewHolder.java

```diff
@@ -20,10 +20,17 @@
     GroupingSettingsViewHolder(View view, GroupingSettings settings) {
         ButterKnife.bind(this, view);
         this.settings = settings;
+        numberOfNamesInList.setText(numberOfNamesInList.getContext().getResources()
```

Comment:
For some reason, getString() has priority in AOSP so it can skip the resources.
```suggestion
        numberOfNamesInList.setText(numberOfNamesInList.getContext().
```